scvi/module/base/_jax_module_wrapper.py

```diff
@@ -40,6 +40,7 @@ class JaxModuleWrapper:
     def __init__(
         self,
         module_cls: JaxBaseModuleClass,
+        is_training: bool = False,
```

Comment:
this doesnt need to be here, just initialize self.is_training to True (pytorch convention)